CHANGELOG.adoc

```diff
@@ -46,7 +46,7 @@ More existing APIs will be migrated to this new common pattern.
 - [releng] Remove the dependency to `reflect-metadata`
 - [releng] Switch to `subscriptions-transport-ws` 0.11.0
 - [releng] Switch to EMF Json 2.3.12
-- https://github.com/eclipse-sirius/sirius-web/issues/3523[#3523] [gantt] Move to @ObeoNetwork/gantt-task-react 0.4.19 to benefit for enhancements
+- https://github.com/eclipse-sirius/sirius-web/issues/3523[#3523] [gantt] Move to @ObeoNetwork/gantt-task-react 0.4.20 to benefit for enhancements
```

Comment:
We are between RC1 and RC2, we are only fixing bugs detected during the homologation. I don't see a point in upgrading gantt-task-react

Reply:
I've removed the unrelated upgrade to `gantt-task-react`